frontend/www/js/omegaup/components/arena/RunSubmit.vue

```diff
@@ -87,6 +91,7 @@ export default class ArenaRunSubmit extends Vue {
   selectedLanguage = '';
   code = '';
   now: number = Date.now();
+  awaitForServerResponse = false;
```

Comment:
nit:

```suggestion
  waitingForServerResponse = false;
```

await significa "esperar" (en infinitivo). entonces esto parece como si se estuviera habilitando algo que espera a que el servidor responda (onda la traducción sería "esperar a que el servidor responda"), en vez de ser la bandera que indica que estamos esperando actualmente (eso casi siempre debería ir en gerundio).

Reply:
Listo